packages/workflow/src/Interfaces.ts

```diff
@@ -2209,6 +2223,42 @@ export type ResourceMapperValue = {
 	matchingColumns: string[];
 	schema: ResourceMapperField[];
 };
+
+export type FilterOperatorType =
+	| 'string'
+	| 'number'
+	| 'boolean'
+	| 'array'
+	| 'object'
+	| 'dateTime'
+	| 'any';
```

Comment:
Where is `any` used?

Reply:
It's used on for example `array:contains`, the right side can be of any type (depending on what's in the array)